pyvista/utilities/common.py

```diff
@@ -0,0 +1,46 @@
+"""Common functions."""
+import collections.abc
+from typing import Sequence, Union
+
+import numpy as np
+
+PointLike = Union[Sequence, np.ndarray]
+
+
+def _coerce_points_like_arg(points: PointLike,
+                            copy: bool=True) -> np.ndarray:
```

Comment:
Naming nitpick: since the name seems to convey "coerce point-like arg", maybe go with `_coerce_pointlike_arg` that also matches the `PointLike` name?